src/game/client/components/menus.h

```diff
@@ -78,7 +78,7 @@ class CMenus : public CComponent
 	char m_aLocalStringHelper[1024];
 
 	int DoButton_DemoPlayer(const void *pID, const char *pText, int Checked, const CUIRect *pRect);
-	int DoButton_Sprite(CButtonContainer *pButtonContainer, int ImageID, int SpriteID, int Checked, const CUIRect *pRect, int Corners);
+	int DoButton_FontIcon(CButtonContainer *pButtonContainer, const char *pText, int Checked, const CUIRect *pRect, int Corners, bool IsDisabled);
```

Comment:
```suggestion
	int DoButton_FontIcon(CButtonContainer *pButtonContainer, const char *pText, int Checked, const CUIRect *pRect, int Corners, bool Enabled = true);
```

Use the positive term and the inverse bool literals.

Reply:
Would I have to set the if statement to !Enabled, or would I have to changed all of the functions to false?